airflow/models.py

```diff
@@ -1638,13 +1646,13 @@ def email_alert(self, exception, is_retry=False):
         task = self.task
         title = "Airflow alert: {self}".format(**locals())
         exception = str(exception).replace('\n', '<br>')
-        try_ = task.retries + 1
+        try_ = self.max_tries
         body = (
             "Try {self.try_number} out of {try_}<br>"
             "Exception:<br>{exception}<br>"
             "Log: <a href='{self.log_url}'>Link</a><br>"
             "Host: {self.hostname}<br>"
-            "Log file: {self.log_filepath}<br>"
+            "Log file directory: {self.log_filedir}<br>"
```

Comment:
I think path is more useful to users (e.g. if you get multiple emails for different try numbers they would have the same directory). Eventually I can see us getting rid of this line though.